### ISSUE TITLE
Website Design

### ISSUE THREAD

**Issue by matteocannavera** [matteocannavera/hello-world#3]
We need to improve the website design. It needs to be more interactive, support more visitors and be more user-friendly.

**Comment by matteocannavera** [comment]
Tyler try to do this

**Comment by ConceptuallyEverAfter** [comment]
I'm on it! I'll work on the design element. We should talk to Todd soon to determine if he will help with the website/software design or not so we can move on and find another person. I have two others in mind.

**Comment by matteocannavera** [comment]
Yeah. I am working on the website too. It's hard to format it though.
Yeah let's give him an ultimatum, if he is not interested we will contact
someone else.

**Comment by ConceptuallyEverAfter** [comment]
Hey I just updated the master code, Take a look.

**Comment by matteocannavera** [comment]
I responded. Check if you can see it. What other functions did you discover
today? Sign up for Slack and we will talk there.
